content/docs/command-reference/add.md

```diff
@@ -151,6 +151,11 @@ not.
   > Note that external outputs typically require an external cache setup. See
   > link above for more details.
 
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace, to
+  [transfer](#example-transfer-to-cache) an external target into cache, and
+  track it in-place. Note that this can be combined with `--to-remote` to avoid
+  storing the data locally, while still adding it to the project.
```

Comment:
```suggestion
- `-o <path>`, `--out <path>` - destination `path` to make a local target copy, or to
  [transfer](#example-transfer-to-cache) an external target into the cache and workspace.
  Note that this can be combined with `--to-remote` to avoid
  storing the data locally, while still adding it to the project.
```